source/common/stats/null_counter.h

```diff
@@ -0,0 +1,33 @@
+#pragma once
```

Comment:
Can you make sure we have coverage for all the null stuff? I think this was the remaining thing in the stats coverage issue? Maybe just close that out as part of this PR?

Reply:
I think that null-stats missing coverage line was fixed in some other PR, and anyway is not evident in https://s3.amazonaws.com/lyft-envoy/coverage/report-master/coverage.html, though several other coverage-gaps in stats exist that are not related to this PR.

Can we address those in a separate PR?

Reply:
OK